### ISSUE TITLE
StarLink Configuration LAN !

### ISSUE THREAD

**Issue by JeedHome44** [Ysurac/openmptcprouter#3232]
Hello !

I wanted to tell you that my StarLink configuration in bypass mode works perfectly !

I just needed to put StarLink in bypass mode and bought a Xiaomi 4A gigabit router and disabled the DHCP !

![LAN OMR StarLink](https://github.com/Ysurac/openmptcprouter/assets/41892950/c538476d-6739-46d6-abaf-69cfb0f5c6e4)

**Comment by Moppel811** [comment]
Please can you give a complete Tutorial, with your exact configuration? 
our Starlink doesn‘t Connect. Not the 192.168.10.1, Not the 100.x.x.x 
Do you have the Bridge Mode on or off ?

**Comment by JeedHome44** [comment]
My OMR ip is always 192.168.100.1

1. so first I put the StarLink in by-pass mode

2. I connected the ethernet port of the StarLink adapter to the WAN port of the Xiaomi 4A Gigabit router

I did wifi tests and I had the StarLink connection on the Xiaomi router.

3. I disabled the DHCP of the Xiaomi router and I connected an Ethernet cable from my LAN to the Xiaomi router

Xiaomi router IP is 192.168.0.1

so I have my WAN 1 which is the Xiaomi router and therefore StarLink and my WAN 2 is my Huawei 4G router

It works perfectly!
